src/app/home/home.component.html

```diff
@@ -62,8 +62,8 @@
   </div>
 </div>
 <!-- Create Space modal -->
-<modal #createSpace title="Create a new collaboration space">
-  <modal-content>
-    <space-dialog></space-dialog>
+<modal #createSpace class="chromeless-modal"  >
```

Comment:
@mitchharpur something we should work out how to do (doesn't need to block this PR) is to put the modal in to the <space-wizard>